src/prep/elaboration-core.k

```diff
@@ -109,14 +109,14 @@ context  elab('Param(_:K,, HOLE,, _:Id))
 */
 rule [elab-Param]:
     <k> elab('Param(K1:K,, T:Type,, X:Id)) => elabRes('Param(K1:K,, T:Type,, X:Id)) ...</k>
-    <elabEnv> ListItem(mapWrap((. => X |-> T) _)) ...</elabEnv>
+    <elabEnv> ListItem(mapWrap((.Map => X |-> T) _)) ...</elabEnv>
 
 //@Removes the last layer from <elabEnv>
 syntax KItem ::= "removeLastElabEnv"
 rule [removeLastElabEnv]:
-    <k> removeLastElabEnv => . ...</k>
-    <elabEnv> ListItem(_) => . ...</elabEnv>
-    <localTypes> ListItem(_) => . ...</localTypes>
+    <k> removeLastElabEnv => .K ...</k>
```

Comment:
Are these changes done for "clarity" or because of the problems with the new parser?